backend/src/main/java/com/handong/rebon/tag/domain/Tag.java

```diff
@@ -39,4 +41,15 @@ private void validateBlankName(String name) {
             throw new TagNameException();
         }
     }
+
+    public void deleteTag() {
+        deleteShopTag();
+        deleteContent();
+    }
+
+    public void deleteShopTag() {
+        this.shopTags.forEach(shopTag -> {
+            shopTag.deleteShopTag();
+        });
+    }
```

Comment:
메서드 참조로 간단하게 나타낼 수 있을 것 같아요
```suggestion
    public void deleteShopTag() {
        this.shopTags.forEach(ShopTag::deleteShopTag);
    }
```

Reply:
좋은 의견 감사합니다!